README.md

```diff
@@ -106,7 +106,7 @@ Currently the core Kedro team consists of:
 * [Yetunde Dada](https://github.com/yetudada)
 * [Ivan Danov](https://github.com/idanov)
 * [Richard Westenra](https://github.com/richardwestenra)
-* [Dmitrii Deriabin](https://github.com/DmitryDeryabin)
+* [Dmitrii Deriabin](https://github.com/DmitriiDeriabinQB)
```

Comment:
I don't know how you caught this but we appreciate it!

Reply:
After getting the readthedocs "Maze" for links to the documentation, I used a Firefox extension to check for more broken links in the README: https://addons.mozilla.org/de/firefox/addon/find-broken-links/

I then did a search&replace for the paths that had changed to catch broken links in other files. Don't know if I got all of them though.
The docs look great, I think you did a great job with the overhaul.
And thanks for this great project!